guides/common/modules/proc_deploying-a-custom-ssl-certificate-to-smart-proxy-server.adoc

```diff
@@ -37,15 +37,15 @@ If the command is successful, it returns two `{certs-generate}` commands, one of
 ----
 Validation succeeded.
 
-To use them inside a NEW {SmartProxyServer}, run this command:
-  {certs-generate} --foreman-proxy-fqdn "{SmartProxyServer}" \
+To use them inside a NEW ${smart-proxy-capitalized}, run this command:
+  {certs-generate} --foreman-proxy-fqdn "{smartproxy-example-com}" \
```

Comment:
Here I'm not sure: the command output is literally `$FOREMAN_PROXY` and not `{smartproxy-example-com}`. So the intended workflow is:
```bash
FOREMAN_PROXY={smartproxy-example-com}
copy_the_suggested_commands_here
```

I'm not sure if this is the best, but the current documentation flow is a bit confusing to me.

Reply:
I have replace this attribute. I would also be open to drop the examples completely because a user _will_ get these commands as stdout.